content/essentials/refs.mdx

```diff
@@ -0,0 +1,106 @@
+# Refs
+
+Refs, or references, allow you to have direct access to DOM element. 
+
+To create a `ref` in Solid, you can get a reference to an element through assigning it to avariable: 
+
+```jsx
+const myDiv = <div>My Element</div>;
+```
+
+For a more idiomatic way, Solid provides the `ref` attribute. 
+Refs in Solid are assignments that occur at creation time before elements are attached to the document DOM.
+To use this, you simply need to declare a variable and use it as a ref attribute:
+
+```tsx
+let myDiv;
```

Comment:
Probably worth pointing out that the TSX way to do this is `let myDiv!: HTMLDivElement;` (we need the `!` because Solid generates an assignment, but ts doesn't know that)